scripts/deploy.sh

```diff
@@ -122,19 +123,28 @@ if [[ ! -f /etc/openstack_deploy/user_osa_secrets.yml ]] && [[ -f /etc/openstack
   mv /etc/openstack_deploy/user_secrets.yml /etc/openstack_deploy/user_osa_secrets.yml
 fi
 
+# set permissions and lay down overrides files
+chmod 0440 /etc/openstack_deploy/user_*_defaults.yml
+if [[ ! -f "$OA_OVERRIDES" ]]; then
+  cp "${RPCD_DIR}"/etc/openstack_deploy/user_osa_variables_overrides.yml $OA_OVERRIDES
+fi
+if [[ ! -f "$RPCD_OVERRIDES" ]]; then
+  cp "${RPCD_DIR}"/etc/openstack_deploy/user_rpco_variables_overrides.yml $RPCD_OVERRIDES
+fi
+
 # Apply host security hardening with openstack-ansible-security
 # The is applied as part of setup-hosts.yml
 if [[ "$DEPLOY_HARDENING" == "yes" ]]; then
```

Comment:
That sounds sensible.  At present `apply_security_hardening` is commented out in user_osa_variables_defaults.yml.  No reason for that not to be uncommented right?